server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -833,4 +861,207 @@ public void testPressureServiceStats() throws Exception {
             });
         }
     }
+
+    public void testScrollCreatedOnReplica() throws Exception {
+        final String primary = internalCluster().startNode();
```

Comment:
nit, simple javadoc the workflow: 

```suggestion
        // create the cluster
        final String primary = internalCluster().startNode();
```